src/content/docs/apis/nerdgraph/examples/configure-infinite-tracing-graphql.mdx

```diff
@@ -159,6 +159,236 @@ Here's the response confirming the change:
+                          "guid": "MXxNT0JJTEV8QVBQTElDQVRJT058Mg",
+                          "name": "My 1st iOS App"
+                        },
+                        "status": "ACTIVE"
+                      },
+                      {
+                        "entity": {
+                          "entityType": "INFRASTRUCTURE_AWS_LAMBDA_FUNCTION_ENTITY",
+                          "guid": "MXxJTkZSQXxOQXwz",
+                          "name": "My 1st Lambda Function"
+                        },
+                        "status": "ACTIVE"
+                      }
+                    ]
+                  },
+                  "endpoints": [
+                    {
+                      "agent": {
+                        "host": "1541eb3c-9cb7-44fc-a764-fa2ab5d507ed.aws-us-east-1.tracing.edge.nr-data.net"
+                      }
+                    }
+                  ],
+                  "id": 259,
+                  "name": "Production services"
+                },
+                {
+                  "dataSourceGroup": {
+                    "dataSources": [
+                      {
+                        "entity": {
+                          "entityType": "BROWSER_APPLICATION_ENTITY",
+                          "guid": "MXxCUk9XU0VSfEFQUExJQ0FUSU9OfDEw",
+                          "name": "My Staging Browser App"
+                        },
+                        "status": "ACTIVE"
+                      },
+                      {
+                        "entity": {
+                          "entityType": "MOBILE_APPLICATION_ENTITY",
+                          "guid": "MXxNT0JJTEV8QVBQTElDQVRJT058MTE",
+                          "name": "My Staging Android App"
+                        },
+                        "status": "ACTIVE"
+                      },
+                      {
+                        "entity": {
+                          "entityType": "INFRASTRUCTURE_AWS_LAMBDA_FUNCTION_ENTITY",
+                          "guid": "MXxJTkZSQXxOQXwxMg",
+                          "name": "My Staging Lambda Function"
+                        },
+                        "status": "ACTIVE"
+                      }
+                    ]
+                  },
+                  "endpoints": [
+                    {
+                      "agent": {
+                        "host": "cd8816a3-b2a2-421b-aa15-5095a056d902.aws-us-west-2.tracing.edge.nr-data.net"
+                      }
+                    }
+                  ],
+                  "id": 632,
+                  "name": "Staging Environment"
+                }
+              ]
+            }
+          }
+        }
+      }
+    }
+  }
+   ```
+   
+4. Execute a mutation using the `id`, one or more entity guids, and whether you would like to ADD, REPLACE, or REMOVE them. For example, to add a new Browser and Mobile app to the `Production services` trace observer with id `259` that we found in the previous step, run the following:
```

Comment:
regarding 'Browser app' or 'Mobile app', the guidelines in https://docs.newrelic.com/docs/style-guide/capitalization/product-capability-feature-usage say we aren't supposed to refer to 'Browser' or 'Mobile' capitalized. I think it would be better to use one of the following approaches: 

• just: browser app or mobile app (lower case) to generally refer to those apps apart from New Relic context, or
• refer to: entities monitored by New Relic's browser monitoring or mobile monitoring (something like that, to give context that we are referring to things from a New Relic perspective.

Reply:
@zuluecho9, I went with your first suggestion.